settings/rest-api.php

```diff
@@ -323,6 +351,51 @@ function register_user_fields(): void {
 		]
 	);
 
+	register_rest_field(
+		'user',
+		'svn_password_required',
+		[
+			'get_callback' => function( $user ) {
+				global $wpdb;
+
+				$user = get_userdata( $user['id'] );
+				if ( ! $user ) {
+					return false;
+				}
+
+				// Committers, supes, etc. It's likely these users will need a SVN password.
+				if ( function_exists( 'is_special_user' ) && is_special_user( $user->ID ) ) {
+					return true;
+				}
+
+				// Plugin committers & Theme authors have this user meta set.
+				if ( $user->has_plugins || $user->has_themes ) {
+					return true;
+				}
+
+				return false;
+			},
+			'schema' => [
+				'type'    => 'boolean',
+				'context' => [ 'edit' ],
+			]
+		]
+	);
```

Comment:
Is this information exposed for unauthenticated users via the `/wp/v2/users/*` endpoints? If so, should we expose them?

Reply:
No, this is only exposed in the `edit` context, which is limited to authenticated users who can edit that given user; which is limited to the actual user and administrators on the given site.